projects/packages/paypal-payments/src/block/edit.js

```diff
@@ -0,0 +1,588 @@
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import {
+	Disabled,
+	ExternalLink,
+	SelectControl,
+	TextareaControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { compose, withInstanceId } from '@wordpress/compose';
+import { dispatch, withSelect } from '@wordpress/data';
+import { useEffect, useRef, useState } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { getWidgetIdFromBlock } from '@wordpress/widgets';
+import clsx from 'clsx';
+import emailValidator from 'email-validator';
+import { get, isEmpty, pick, trimEnd } from 'lodash';
```

Comment:
Since you're using those external tools, you'd need to bring them as dependencies in the package's `package.json` file.

Reply:
Added `email-validator` dependency in [515815e](https://github.com/Automattic/jetpack/pull/43413/commits/515815e86ee65b6388567ec3e481062a85966c84).